wct-core/src/main/java/org/webcurator/core/profiles/PolitenessOptions.java

```diff
@@ -0,0 +1,127 @@
+package org.webcurator.core.profiles;
+
+public class PolitenessOptions {
+    private double delayFactor;
+    private long minDelayMs;
+    private long maxDelayMs;
+    private long respectCrawlDelayUpToSeconds;
+    private long maxPerHostBandwidthUsageKbSec;
+    public static final PolitenessOptions POLITE_OPTIONS = new PolitenessOptions(10.0d, 9000, 90000, 900, 400);
+    public static final PolitenessOptions MEDIUM_OPTIONS = new PolitenessOptions(5.0d, 3000, 30000, 300, 800);
```

Comment:
@ahamblyn Could these politeness defaults be configurable via spring? Just trying to think of a way they could be updated in the future without requiring a code change.